connector/connect/client/jvm/src/test/scala/org/apache/spark/sql/CheckpointSuite.scala

```diff
@@ -50,12 +51,20 @@ class CheckpointSuite extends ConnectFunSuite with RemoteSparkSession with SQLHe
     checkFragments(captureStdOut(block), fragmentsToCheck)
   }
 
-  test("checkpoint") {
+  test("localCheckpoint") {
```

Comment:
note: there are no tests that test Connect reliable checkpoint. I renamed this test accordingly.